src/style.css

```diff
@@ -0,0 +1,66 @@
+@font-face {
+  font-family: Avenir;
+  src: url(fonts\Avenir-Book.ttf);
+  font-style: normal;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(fonts\Avenir-Heavy.ttf);
+  font-style: bold;
+}
+
+html {
+  font-family: inherit;
+  box-sizing: border-box;
+  color: #3e4e61;
+  font-weight: 300;
+  font-style: normal;
+}
+
+* {
+  box-sizing: border-box;
+  color: #3e4e61;
+}
+
+.wrapper {
+  margin-top: 20px;
+}
+
+.searchbar {
+  background-image: url(images\Search.svg);
+  background-repeat: no-repeat;
+  border: 1px solid #e1e7ed;
+  box-sizing: border-box;
+  box-shadow: 0 1px 8px rgb(61 78 97 / 10%);
+  border-radius: 4px;
+  outline: none;
+}
+
+.big {
+  width: 100%;
+  height: 70px;
+  font-size: 16px;
+  padding: 23px 20px 23px 62px;
+  background-position: left 25px bottom 25px;
+  background-size: 19px;
+}
+
+.small {
+  width: 100%;
+  height: 42px;
+  font-size: 14px;
+  padding: 10px 20px 11px 33px;
```

Comment:
Use only padding-left and line-height for aligning text by center (should fix tests)